tests/testthat/test-user_utils.R

```diff
@@ -1,13 +1,15 @@
 # convert_blanks_to_na ----
 ## Test 1: blank strings are turned into `NA` ----
-test_that("convert_blanks_to_na Test 1: blank strings are turned into `NA`", {
+# ---- user_utils, test 1: convert_blanks_to_na Test 1: blank strings are turned into `NA` ----
```

Comment:
It seems something went wrong here. Could you fix it? Usually calling the "Format test_that test file" addin should produce the correct comments and descriptions.

Reply:
these comments and descriptions were produced using the **Format test_that test file** and I have to fix the issue above but missed that one. I noticed there are a few more test programs that aren't formatted using the agreed format so I will will **Format test_that test file** to make sure there isn't a bug.